internal/examples/thrift-hello/hello/main.go

```diff
@@ -37,60 +36,52 @@ import (
 //go:generate thriftrw --plugin=yarpc echo.thrift
 
 func main() {
-	httpTransport := http.NewTransport()
+	// configure a dispatcher for the service "hello",
+	// expose the service over an HTTP inbound on port 8086,
+	// and configure outbound calls to service "hello" over HTTP port 8086 as well
 	dispatcher := yarpc.NewDispatcher(yarpc.Config{
 		Name: "hello",
 		Inbounds: yarpc.Inbounds{
-			httpTransport.NewInbound(":8086"),
+			http.NewTransport().NewInbound(":8086"),
 		},
 		Outbounds: yarpc.Outbounds{
 			"hello": {
-				Unary: httpTransport.NewSingleOutbound("http://127.0.0.1:8086"),
+				Unary: http.NewTransport().NewSingleOutbound("http://127.0.0.1:8086"),
```

Comment:
You might be right - I was trying to get the example to *start* with a line that instantiates the dispatcher